mavros/src/plugins/global_position.cpp

```diff
@@ -68,8 +68,7 @@ class GlobalPositionPlugin : public MavRosPlugin {
 
 		// fused global position
 		gp_fix_pub = gp_nh.advertise<sensor_msgs::NavSatFix>("global", 10);
-		gp_pos_pub = gp_nh.advertise<geometry_msgs::PoseWithCovarianceStamped>("local", 10);
-		gp_vel_pub = gp_nh.advertise<geometry_msgs::TwistStamped>("gp_vel", 10);
+		gp_odom_pub = gp_nh.advertise<nav_msgs::Odometry>("local", 10);
```

Comment:
Well turns out to gives us local position. This is just to have congruence with `global`, which gives the fix topic. What do you think?